Added a note for Shortname Support Policy (#1300)
theforeman/foreman-documentation: guides/common/modules/ref_system-requirements.adoc

* A system umask of 0022
* Full forward and reverse DNS resolution using a fully-qualified domain name

> NOTE: The shortnames in the hostnames are no longer supported for the {Project} and {SmartProxy} server.

Reviewer: I think it should be "Project does not support using shortnames" instead of "no longer", because we deliver versioned documentation. This means that users always get the appropriate docs for their version of Foreman.

cc @melcorr 

For the note box, I am fine with it, but maybe you can reword it as a requirements rather then "warning".

Reviewer 2: At the rate people are trying to add notes to our docs, we would have only notes. 
It reads to me like a regular sentence. Let's remove the NOTE prefix. 
I agree with removing the no longer for the reasons @maximiliankolb outlined.

Author: Noted 👍

Author: Modified.

> Also, the custom certificate's Common Name (CN) must have a fully qualified domain name (FQDN) instead of a shortname.

Reviewer: Please reword to make it active and "Common Name within/of the custom certificate". I am unsure about capitalication.

Reviewer 3: ```suggestion
When using custom certificates, the Common Name (CN) of the custom certificate must be a fully qualified domain name (FQDN) instead of a shortname.
```

> Also, the custom certificate's Common Name (CN) must have a fully qualified domain name (FQDN) instead of shortnames.

Reviewer 2: ```suggestion
Also, the custom certificate's Common Name (CN) must have a fully qualified domain name (FQDN) instead of a shortname.
```

Author: Modified.

> This does not apply to the clients of a {Project}.

Reviewer 3: I know the goal of this statement is to head off getting this question, which we get every time we talk about shortname support. Users hear about this and ask "does this also apply to clients", so while it seems obvious it should not since we did not statement I am inclined towards keeping this statement to head off unnecessary back and forth.

> This does NOT apply to the clients of a {Project} and they can continue to use shortnames if required by the user.

Reviewer 2: Never use caplocks. 
We don't use an article before {Project}
```suggestion
This does not apply to the clients of {Project}.
```
Minimalism.

Reviewer 2: We talk to the user, not about the user.

Author: Noted, Melanie. Taking changes 👍

Author: Modified.
